src/info/info.nim

```diff
@@ -0,0 +1,72 @@
+import std/[algorithm, json, os, sets, terminal]
+import ".."/cli
+
+proc parseTrackConfig(trackDir: string): JsonNode =
```

Comment:
We will need this in other places too (e.g. for linting), so it might be nice to have it in a shared module. We can do that later of course.

Reply:
Now going via `jsony`. I might want to move the types out of `lint/track_config` at some point.